.github/workflows/build.yml

```diff
@@ -45,14 +45,22 @@ jobs:
             *.cache-from=type=gha,scope=validate
             *.cache-to=type=gha,scope=validate,mode=max
 
-      # Disabled netlify-deploy due to flakey 502 http errors
-      # - name: copy static files
-      #   if: github.event.pull_request.head.repo.fork == false
-      #   run: docker run -v ${PWD}:/output documentation:latest cp -r /usr/share/nginx/html /output/_site
-      # - uses: ./.github/actions/netlify-deploy
-      #   if: github.event.pull_request.head.repo.fork == false
-      #   with:
-      #     directory: _site
-      #     netlify_token: ${{ secrets.NETLIFY_AUTH_TOKEN }}
-      #     netlify_account_slug: ${{ secrets.NETLIFY_ACCOUNT_SLUG }}
-      #     site_name: "${{ github.repository }}/${{ github.head_ref }}"
```

Comment:
With this refactoring we might enable this workflow and see if it works fine now but don't think it's necessary to have it anymore if we want to only accept pull requests from forks and disable auto publish. Step would look like this:

```yaml
      -
        name: Netlify deploy
        if: github.event.pull_request.head.repo.fork == false
        uses: docker/bake-action@v2
        with:
          targets: netlify-deploy
        env:
          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
          NETLIFY_ACCOUNT_SLUG: ${{ secrets.NETLIFY_ACCOUNT_SLUG }}
          NETLIFY_SITE_NAME: ${{ github.repository }}/${{ github.head_ref }}
          NETLIFY_PUBLISH_DIR: "_site"
```